domain/src/main/java/edu/nextstep/camp/calculator/domain/CalculateHistory.kt

```diff
@@ -0,0 +1,11 @@
+package edu.nextstep.camp.calculator.domain
+
+class CalculateHistory {
+    val calculateHistories: List<CalculateResult>
+        get() = _calculateHistories
```

Comment:
방어적 복사에 대한 개념을 찾아보시고, 이 프로퍼티를 반환하는 방법이 문제가 없을지 알려주세요.

Reply:
다른 객체에서 calculateHistories를 참조하고 수정하게되면 _calculateHistories의 내용물이 변경되게되네요..!
```suggestion
    val calculateHistories: List<CalculateResult>
        get() = _calculateHistories.toList()
```
의 형식으로 수정 진행했습니다! 감사합니다 :)